### PR TITLE
Remove randomness from condition example

### PR DESCRIPTION
**Pull request by drewbutlerbb4**
The unit test for dsl.Condition previously used a random number generator to dictate which event occurred and therefore which condition would execute. This causes issues with consistent testing as there are multiple valid potential outputs. This test has been changed to be deterministic, to address this issue, by using pipeline parameters to dictate which event occurred and setting default values for the pipeline.

### CODE REVIEW

**Comment by drewbutlerbb4** [comment]
Thanks @ckadner, I have addressed your comment.

File: sdk/python/tests/compiler/testdata/condition.py [hunk 1]
```diff
@@ -40,17 +40,14 @@ def __init__(self, name, msg):
     name='Flip Coin Example Pipeline',
     description='Shows how to use dsl.Condition.'
 )
-def flipcoin():
-    flip = FlipCoinOp('flip')
```
[the remark this answers is not in the document]
Comment (drewbutlerbb4) [reply sdk/python/tests/compiler/testdata/condition.py:44]: Resolving because the flip coin class is being used now

**Comment by animeshsingh** [comment]
Can we merge this first, before merging @ckadner e-2-e test with conditional tests enabled

cc @drewbutlerbb4 @Tomcli

**Pull request merged**